website/index.html

```diff
@@ -21,87 +21,94 @@ <h2>
             </a>
         </header>
         <div class="flex-all-content">
+            <div class="gradient-for-line">
+            </div>
             <div class="line">
                 <div></div>
             </div>
-            <div id="main-content">
+            <div class="main-content">
                 <div class="text-image-flex">
                     <div class="introduction width-mobile">
                         <p>
                             <span>Hi! I am a developer</span>
-                            <br>
-                            <br>
+                            <br/>
+                            <br/>
                             I am a front end and 
-                            back end developer, that creates technical solutions for <br>
+                            back end developer, that creates technical solutions for <br/>
                             companies/clients of all genres.
-                            <br>
-                            <br>
+                            <br/>
+                            <br/>
                             Want to see some of my work? 
-                            <br>
+                            <br/>
                             ...You are looking at it!
                         </p>
                     </div>
                     <div class="image-container-3">
```

Comment:
This change will let me know what image im am working with if i am only looking at the css. If i have a number i easily forget which image 3 is.
```suggestion
                    <div class="portraitImg">
```

Reply:
6edbe52

Reply:
Nitpick: I would think the element is an img if i saw the css only. Maybe `portrait-image-container`?